### PR TITLE
Add recaptcha V3

### PR DESCRIPTION
**Pull request by piotrze**

### CODE REVIEW

**Review by diana-lakatos — APPROVED** [review]
Reviewed and edited. Thank you!

**Comment by diana-lakatos** [comment]
Closes #513

File: marketplace_builder/views/pages/tutorials/forms/form-configuration.liquid [hunk 1]
```diff
@@ -206,6 +206,15 @@ This is very useful when you want to restrict user permissions to modify only re
...
 
 ### spam_protection
-String, type of spam protection that will be used. Needs to be configured in Partner Portal. A separate topic explains how to add [reCaptcha Spam Protection](/tutorials/integrations/adding-recaptcha-spam-protection).
+Object, strategy and configuration for strategy.
+
+```yaml
+spam_protection:
+  strategy: "recaptcha_v2" # required, possible values: recaptcha_v2, recaptcha_v3
+  action: "signup"         # optional, action for recaptcha V3
+  minimum_score: 0.7       # optional, minimum score that we accept for recaptcha V3
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/forms/form-configuration.liquid]:
```suggestion
  minimum_score: 0.7       # optional, minimum score that we accept for recaptcha V3, default: 0.5
```

File: marketplace_builder/views/pages/tutorials/forms/form-configuration.liquid [hunk 2]
```diff
@@ -206,6 +206,15 @@ This is very useful when you want to restrict user permissions to modify only re
 Array of short text message notification names that will be executed after the form is successfully submitted.
 
 ### spam_protection
-String, type of spam protection that will be used. Needs to be configured in Partner Portal. A separate topic explains how to add [reCaptcha Spam Protection](/tutorials/integrations/adding-recaptcha-spam-protection).
+Object, strategy and configuration for strategy.
+
+```yaml
+spam_protection:
+  strategy: "recaptcha_v2" # required, possible values: recaptcha_v2, recaptcha_v3
+  action: "signup"         # optional, action for recaptcha V3
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/forms/form-configuration.liquid]:
```suggestion
  action: signup         # optional, action for recaptcha V3
```

File: marketplace_builder/views/pages/tutorials/forms/form-configuration.liquid [hunk 3]
```diff
@@ -206,6 +206,15 @@ This is very useful when you want to restrict user permissions to modify only re
 Array of short text message notification names that will be executed after the form is successfully submitted.
 
 ### spam_protection
-String, type of spam protection that will be used. Needs to be configured in Partner Portal. A separate topic explains how to add [reCaptcha Spam Protection](/tutorials/integrations/adding-recaptcha-spam-protection).
+Object, strategy and configuration for strategy.
+
+```yaml
+spam_protection:
+  strategy: "recaptcha_v2" # required, possible values: recaptcha_v2, recaptcha_v3
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/forms/form-configuration.liquid]:
```suggestion
  strategy: recaptcha_v2 # required, possible values: recaptcha_v2, recaptcha_v3
```

**Comment by piotrze** [comment]
Hold on with merging until feature is deployed on production!

**Pull request merged**